gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceSettingsClassComposer.java

```diff
@@ -270,40 +278,44 @@ private static List<MethodDefinition> createSettingsGetterMethods(
     List<MethodDefinition> javaMethods = new ArrayList<>();
     for (Method protoMethod : service.methods()) {
       String javaStyleName = JavaStyle.toLowerCamelCase(protoMethod.name());
-      String javaMethodName =
-          String.format("%sSettings", JavaStyle.toLowerCamelCase(protoMethod.name()));
+      String javaMethodName = String.format("%sSettings", javaStyleName);
       MethodDefinition.Builder methodBuilder =
           methodMakerFn.apply(getCallSettingsType(protoMethod, typeStore), javaMethodName);
-      javaMethods.add(
-          methodBuilder
-              .setHeaderCommentStatements(
-                  SettingsCommentComposer.createCallSettingsGetterComment(
-                      getMethodNameFromSettingsVarName(javaMethodName), protoMethod.isDeprecated()))
-              .setAnnotations(
-                  protoMethod.isDeprecated()
-                      ? Arrays.asList(AnnotationNode.withType(TypeNode.DEPRECATED))
-                      : Collections.emptyList())
-              .build());
+      javaMethods.add(methodBuilderHelper(protoMethod, methodBuilder, javaMethodName));
       if (protoMethod.hasLro()) {
         javaMethodName = String.format("%sOperationSettings", javaStyleName);
```

Comment:
nit: there is no need to update and reuse this string variable. It took me a second to realize the difference of L284 vs L289